tests/contrib/httpx/test_httpx.py

```diff
@@ -0,0 +1,207 @@
+import contextlib
+import os
+
+import httpx
+import pytest
+
+from ddtrace import config
+from ddtrace.contrib.httpx.patch import patch
+from ddtrace.contrib.httpx.patch import unpatch
+from ddtrace.settings.http import HttpConfig
+from tests.utils import override_config
+from tests.utils import override_http_config
+from tests.utils import snapshot_context
+
+
+# host:port of httpbin_local container
+HOST = "localhost"
+PORT = 8001
+
+
+def get_url(path):
+    # type: (str) -> str
+    return "http://{}:{}{}".format(HOST, PORT, path)
+
+
+@pytest.fixture(autouse=True)
+def patch_httpx():
+    patch()
+    try:
+        yield
+    finally:
+        unpatch()
+
+
+@contextlib.contextmanager
+def _snapshot_context(request):
+    token = ".".join(
+        [
+            request.node.module.__name__,
+            request.node.name,
+        ]
+    )
+    with snapshot_context(token=token):
+        yield
+
+
+@pytest.mark.asyncio
+async def test_get_200(request):
+    url = get_url("/status/200")
+    with _snapshot_context(request):
+        resp = httpx.get(url)
+        assert resp.status_code == 200
+
+    with _snapshot_context(request):
```

Comment:
Are we expecting 2 spans to be generated per test since we're testing synchronous and async tracing per test?